titiler/models/metadata.py

```diff
@@ -22,14 +22,14 @@ class cogInfo(BaseModel):
     """COG Info."""
 
     bounds: Tuple[float, float, float, float]
-    band_metadata: List[Tuple[int, Dict[int, Any]]]
     band_descriptions: List[Tuple[int, str]]
     dtype: str
     colorinterp: List[str]
     nodata_type: NodataTypes
     scale: Optional[float]
     offsets: Optional[float]
     colormap: Optional[Dict[int, ColorTuple]]
+    band_metadata: Optional[List[Tuple[int, Dict[int, Any]]]]
```

Comment:
Switched to optional to allow reuse for `/mosaicjson/info` response model -- I don't think band metadata makes sense for mosaicjson metadata.  Should maybe write a `mosaicInfo` model instead and revert this change.